src/task2.1.js

```diff
@@ -0,0 +1,18 @@
+import fs from 'fs';
+import path from 'path';
+import csv from 'csvtojson';
+
+const filePath = path.join(__dirname, '../csv', 'nodejs-hw1-ex1.csv');
+const outputPath = path.join(__dirname, '../output', 'task-2.1.json');
+
+(async function main() {
+  const fileContent = fs.readFileSync(filePath, 'utf8');
+  const jsonObj = await csv().fromString(fileContent);
+
+  fs.writeFileSync(outputPath, JSON.stringify(jsonObj), (err) => {
```

Comment:
if you use sync method, callback is not needed

also, sync methods are not recommended to use in production applications, it's better to use fs promise api
https://nodejs.org/api/fs.html#promises-api

Reply:
Changed.